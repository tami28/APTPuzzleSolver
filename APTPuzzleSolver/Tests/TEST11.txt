NumElements=25
10 0 0 1 0
1 -1 0 0 -1
25 0 0 -1 -1
18 1 0 -1 -1
7 1 0 0 1
23 0 -1 0 1
5 0 -1 0 0
12 0 -1 -1 -1
11 1 1 -1 -1
19 1 1 0 -1
2 0 -1 1 0
4 -1 0 0 0
21 0 1 0 0
9 0 1 0 0
3 0 1 0 0
20 0 0 1 0
14 -1 1 -1 0
17 1 1 -1 -1
8 1 1 1 -1
6 -1 -1 0 0
16 0 0 0 1
13 0 0 -1 1
15 1 1 0 1
24 0 1 1 -1
22 -1 0 0 -1