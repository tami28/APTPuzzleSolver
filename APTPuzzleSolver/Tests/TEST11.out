10, 1, 25, 18, 7
20, 14, 17, 8, 6,
16, 13, 15, 24, 22
23, 5, 12, 11, 19
2, 4, 21, 9, 3